components/sso-hostobject/org.wso2.carbon.hostobjects.sso/pom.xml

```diff
@@ -67,6 +67,11 @@
             <artifactId>commons-lang</artifactId>
             <version>2.6.0.wso2v1</version>
         </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+            <version>2.2.3</version>
```

Comment:
dependencies version needs to mentioned in the parent pom of the component, and the actual components need to inherit from there. We avoid adding new dependency in the component level, and try to use same dependency version throughout the repo.